### DISCUSSION TITLE
Georgia lite?

### DISCUSSION THREAD

**Discussion by TheMan133** [TT-ReBORN/Georgia-ReBORN#139] [Q&A] [answered]
Hello. I love your theme but the thing which very much postponed me using your theme was that you have to install custom fonts in order to use theme (as far as I know they may cause your computer to lag since they will preload on every windows startup). I think I'm not alone. Could you please make a release without necessity of custom fonts, if it's not hard?

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @TheMan133,

it is not possible to remove all custom fonts, they are needed for symbols ( compact menu, minimize, fullscreen, close, 
and all lower transport buttons, library and biography buttons etc.) in the theme. 
You can only remove all the Open Sans custom fonts if you want to:
```
OpenSans-SemiBoldItalic.ttf
OpenSans-Bold.ttf
OpenSans-BoldItalic.ttf
OpenSans-ExtraBold.ttf
OpenSans-ExtraBoldItalic.ttf
OpenSans-Italic.ttf
OpenSans-Light.ttf
OpenSans-LightItalic.ttf
OpenSans-Medium.ttf
OpenSans-MediumItalic.ttf
OpenSans-Regular.ttf
OpenSans-SemiBold.ttf
```

There are only 24 fonts to install, this will NOT slow down your computer, i.e have longer Windows startup.
Maybe only if you have thousands of fonts installed, the Windows boot process will take a few seconds longer.
Everyone nowadays is using fast SSD's, there is absolutely no concern about performance issues.

-TT

**Reply by TheMan133** [reply]
I see, ty
